torch/csrc/jit/serialization/export.cpp

```diff
@@ -558,7 +559,12 @@ void GraphEncoder::EncodeValueInfoType(
           }
         } else if (sizes[i].is_static()) {
           shape->mutable_dim(i)->set_dim_value(sizes[i].static_size());
-        } else {
+        } else if (!is_sequence_tensor) {
```

Comment:
Does the code on line 556-559 need to be guarded by this bool as well?
If not, could you please explain the difference?

Reply:
No, because dynamic axis symbol from 556-559 are found directly in `dynamic_axes` argument. This implies this sequence is graph output, and users explicitly assigned that symbol.